packages/EasyBatch/src/Serializer/Serializer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyBatch\Serializer;
+
+use EonX\EasyBatch\Interfaces\SerializerInterface;
+
+class Serializer implements SerializerInterface
+{
```

Comment:
Can't we name it more specifically? I mean `SomeSpecificSerializer` instead of generic `Serializer`.

Reply:
Done.